tests/e2e/adu/device/e2e_device_commands.c

```diff
@@ -150,16 +151,60 @@ static uint8_t ucAzureIoTADURootKeyN[ 385 ]
     0x7e, 0x40, 0x48, 0xa9, 0x12, 0x01, 0xa8, 0xd9, 0x7e, 0xf3, 0xa5, 0x1b, 0xf1, 0xfb, 0x90,
     0x77, 0x3e, 0x40, 0x87, 0x18, 0xc9, 0xab, 0xd9, 0xf7, 0x79
     };
-static uint8_t ucAzureIoTADURootKeyE[ 3 ] = { 0x01, 0x00, 0x01 };
+static uint8_t ucAzureIoTADURootKeyE200702[ 3 ] = { 0x01, 0x00, 0x01 };

```

Comment:
so why specify size? vs below? Compat issue??

static uint8_t ucAzureIoTADURootKeyE200702[] = { 0x01, 0x00, 0x01 };

Reply:
I think it was mostly to make sure we have them there correctly (size is as it should).